Plotting_details.md

```diff
@@ -0,0 +1,14 @@
+## Plots and Charts
+
+ * Single line plots
+ * Multiple line plots
+ * Legends
+ * Colorbars
+ * Panel plots
+ * Map projections
+ * Contour plots
+ * Wind plots 
+ * Seaborn: (To be completed)
+ * Importing data into GIS (To be completed)
+
+[Link to training](???)
```

Comment:
It seems a bit long for me. Should it be split?

Reply:
Aren't we currently just looking at what skills are needed?

Reply:
Well, we are also trying to finish this. So I also like to look at it with the end goal in mind. And one of the goal is to inform us on what training to develop and what should be covered in this training. So the idea is to look at what skills are needed but to decide how to organise those skills into logical training blocks. 1 block is not necessarily a 1 hour training session, it can span several hours. But still, the skills should be organised per training. I'm happy if we decide this one block is a logic block. Although, I'm wondering if GIS stuff could be separate (as being more specific) and maybe the same with Seaborn. The same we could separate plots and maps, although working with axes, color etc can be common skills.